marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils/invoke-queue-write.sjs

```diff
@@ -88,6 +88,15 @@ for (let content of contentArray) {
     deleteContent(content, temporalCollection);
   } else {
     const metadata = context.metadata;
+    const quality = context.quality || 0;
+
+    if (!Number.isInteger(quality)) {
```

Comment:
I think if(!Number(quality)) is sufficient , other checks can be removed. Also the error message can provide more information like doc uri. Something like `Unable to insert document: ${content.uri}; cause: invalid value for 'quality' : String(quality)`

Reply:
I actually don't think we need any code here; ML will throw an error itself. We should see what that error looks like before writing any code. Otherwise, we're getting into the business of duplicating the logic that ML already enforces on valid values for quality.